static/styles/app_components.css

```diff
@@ -542,6 +542,7 @@ div.overlay {
 
 .white_zulip_icon_without_text {
     display: inline-block;
+    /* This sadly does not have an easy way to use the CDN. */
     background-image: url(../../static/images/logo/white-zulip-logo-without-text.svg);
```

Comment:
Webpack already compiles this to `background-image: url("files/white-zulip-logo-without-text.16fe7644bab4c1da60144b86cb9330b4.svg")`.

(The extra `../static/` in the relative path could be simplified away, though.)